app/src/main/java/org/mozilla/fenix/utils/ClipboardHandler.kt

```diff
@@ -19,12 +22,13 @@ class ClipboardHandler(context: Context) {
 
     var text: String?
         get() {
-            if (clipboard.isPrimaryClipEmpty() ||
-                !clipboard.isPrimaryClipPlainText()) {
-                return null
-            }
+            if( !clipboard.isPrimaryClipEmpty() &&
+                (clipboard.isPrimaryClipPlainText() or
```

Comment:
`or` corresponds to bit-wise or (`|`), did you mean to use `||` here?

Reply:
`or` refers to bit-wise or. Im not sure if Kotlin uses (  |  ), since I've mostly used the former.

Reply:
We generally use the logical operators. Could we switch to using `||` here for consistency?